src/app/inventory/store/d2-item-factory.ts

```diff
@@ -236,7 +236,7 @@ export function makeItem(
   item: DestinyItemComponent,
   /** the ID of the owning store - can be undefined for fake collections items */
   owner: DimStore | undefined
-): DimItem | null {
+): DimItem | undefined {
```

Comment:
I've never particularly like using `null`, so this switches from functions that can return `null` for items to `undefined`.